### DISCUSSION TITLE
Get this error on steamdeck with boxtron

### DISCUSSION THREAD

**Discussion by knoxvillesjoker** [DavidoTek/ProtonUp-Qt#129] [Q&A]
dosbox:missing
inotify-tools: missing
timidity missing

You may need to roll in those repos to the package to install.  

Straight dosbox-staging works fine.

Steamdeck runs KDE Plasma with a Proton/vulkan overlay.

**Comment by DavidoTek** [comment]
Yes, Boxtron has a few dependencies.

You can install the Flatpak version of DosBox Staging straight from the Discover app store, that doesn't require you to enable dev mode on the Steam Deck afaik. You can check out this guide (not tested): https://www.reddit.com/r/SteamDeck/comments/txjtgr/comment/ig5w96w
